lib/msal-common/src/telemetry/performance/PerformanceEvent.ts

```diff
@@ -531,6 +543,11 @@ export const PerformanceEventAbbreviations: ReadonlyMap<string, string> =
         ],
         [PerformanceEvents.Sha256Digest, "Sha256Digest"],
         [PerformanceEvents.GetRandomValues, "GetRandomValues"],
+        [PerformanceEvents.GenerateHKDF, "genHKDF"],
+        [PerformanceEvents.GenerateBaseKey, "genBaseKey"],
+        [PerformanceEvents.Base64Decode, "b64Decode"],
+        [PerformanceEvents.UrlEncodeArr, "urlEncArr"],
+        [PerformanceEvents.Encrypt, "encrypt"],
```

Comment:
Corresponding decrypt?

Reply:
Decrypt is wrapped up in the `importExistingCache` event. We unfortunately don't have a good way to capture data for events that occur more than once within a single row so we'll capture the perf all up + the count of items decrypted and can average it that way. Now that you mention it though we have a similar problem with encrypt so I may need to rethink how we measure that one...

Reply:
@konstantin-msft If you have thoughts here I'd love your input

Reply:
I actually did end up adding decrypt because it gives us the call count built in, but we'll still want to close on a good way to interpret the data for events like this that happen a handful of times per call

Reply:
Besides the number of function calls captured by telemetry wrappers these calls will be captured in the "context" field as "decrypt_1", "decrypt_2", etc. It should be enough for perf analysis and debugging.